habitat-lab/habitat/datasets/rearrange/run_episode_generator.py

```diff
@@ -199,6 +210,33 @@ class RearrangeEpisodeGeneratorConfig:
     #  }
     markers: List[Any] = field(default_factory=list)
 
+    # If we want to re-generate the nav mesh or not
+    regenerate_new_mesh: bool = True
+    # The radius of the agent in meters
+    agent_radius: float = 0.25
+    # The height of the agent in meters
+    agent_height: float = 0.61
+    # The max climb of the agent
+    agent_max_climb: float = 0.02
+    # The maximum slope that is considered walkable in degrees
+    agent_max_slope: float = 5.0
```

Comment:
Note that this parameter is not the default value and is not set in training configs.
However, since this generator parameter setting is stricter, it should affect training in any way.